src/screens/Ticketing/Purchase/utils.ts

```diff
@@ -29,3 +31,27 @@ export const getPurchaseFlow = (
     };
   }
 };
+
+export function getExpireDate(iso: string): string {
+  let date = parseISO(iso);
+  let year = date.getFullYear();
+  let month = date.getMonth();
+  if (month === 0) {
+    month = 12;
+    year--;
+  }
+  return `${month < 10 ? '0' + month : month}/${year.toString().slice(2, 4)}`;
```

Comment:
Wouldn't it be easier to just use date-fns here to avoid padding and date oddities?

Reply:
didn't really consider the implementation. I merely moved the function from component to utils.
That said, I agree.

Reply:
This is the perfect time for [boy/girl scout driven development](https://dzone.com/articles/the-boy-scout-software-development-principle)!